tests/alerts/test_guard_duty_probe.py

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+from positive_alert_test_case import PositiveAlertTestCase
+from negative_alert_test_case import NegativeAlertTestCase
+from alert_test_suite import AlertTestSuite
+
+
+class TestGuardDutyProbe(AlertTestSuite):
+    alert_filename = "guard_duty_probe"
+    alert_classname = "AlertGuardDutyProbe"
+
+    # This event is the default positive event that will cause the
+    # alert to trigger
+    default_event = {
+        "_type": "event",
+        "_source": {
+            "source": "guardduty",
+            "details": {
+                "sourceipaddress": "1.2.3.4",
+                "finding": {"action":
+                            {"actionType": "PORT_PROBE"}
+                            }
```

Comment:
Funny indent

Reply:
Can you just fix this in the github editor?

Reply:
Done